packages/wrappers/src/base.ts

```
@@ -112,19 +111,7 @@ export class BaseWrapper {
-        }
-        const dispatcher = new ProxyAgent(Settings.ADDON_PROXY);
-        console.error(
-          `|ERR| wrappers > base > ${this.addonName}: Got error: ${error.message} when fetching from ${sanitisedUrl}, trying with proxy instead`
-        );
-        response = await uFetch(url, {
-          dispatcher,
-          method: 'GET',
-          headers: headers,
-          signal: controller.signal,
-        });
+throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)